### PR TITLE
[MM-47300] Migrate "/enterprise/elasticsearch_autocomplete e2e" tests to TypeScript

### PR DESCRIPTION
**Pull request by cyberbuff**
#### Summary

Converting following spec files under /integration/enterprise/elasticsearch_autocomplete to Typescript
1. channels_spec.js
2. renaming_spec.js
3. users_in_channel_switcher_spec.js
4. channels_with_special_characters_spec.js
5. renaming_team_spec.js
6. users_in_message_input_box_spec.js
7. system_console_spec.js
8. users_spec.js

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/21308

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: e2e/cypress/tests/integration/enterprise/elasticsearch_autocomplete/helpers/index.ts [hunk 1]
```diff
@@ -0,0 +1,260 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```
Comment (cyberbuff) [inline e2e/cypress/tests/integration/enterprise/elasticsearch_autocomplete/helpers/index.ts:1]: Even though I renamed the file, git considers it as a new file for some reason. If you have any suggestions to fix this, kindly let me know

**Comment by saturninoabril** [comment]
@cyberbuff Sorry for the delay. Could you please help update and fix conflict? I'll take a look after. Thanks!

**Comment by lieut-data** [comment]
Heads up that as part of our efforts to move to a monorepo, we're closing out a number of older pull requests like this one to help streamline the effort.

If you'd like to preserve these changes -- even if you're not the original author! -- feel free to resubmit the pull request against the monorepo once it's ready. You can subscribe to [mattermost-server-issue-22420](https://www.github.com/mattermost/mattermost-server/issues/22420) for status updates on this effort.

**Pull request closed without merging**